### PR TITLE
Clarify how to update iai-callgrind-runner.

### PR DESCRIPTION
**Pull request by nihohit**
It confused me (https://github.com/Joining7943/iai-callgrind/issues/21), so maybe others will be confused too.

### CODE REVIEW

**Comment by Joining7943** [comment]
Thanks for contributing! The pr looks good. It just needs a few corrections.

**Comment by nihohit** [comment]
@Joining7943. What are the corrections?

**Comment by Joining7943** [comment]
@nihohit See the review for the lines which need correction. Maybe something like `Please update iai-callgrind to '{}' in your Cargo.toml file`

**Comment by nihohit** [comment]
Sorry, I don't see the review comments - maybe you didn't post it?

**Review by Joining7943 — CHANGES_REQUESTED** [review]

**Comment by Joining7943** [comment]
yeah. Forgot to submit the reviews. Hope you see them now

**Comment by Joining7943** [comment]
Looks good now! There's just a small formatting issue left. This repo uses `cargo +nightly fmt` for formatting (and just for the formatting).

File: iai-callgrind-runner/src/main.rs [hunk 1]
```diff
@@ -70,19 +70,20 @@ fn main() {
...
+                            "iai-callgrind-runner ({0}) is older than iai-callgrind ({1}). Please \
+                             update iai-callgrind-runner by calling 'cargo install --version {1} \
+                             iai-callgrind-runner'",
                             runner_version, library_version
                         ),
                         Cmp::Gt => error!(
-                            "iai-callgrind-runner ({}) is newer than iai-callgrind ({}). Please \
-                             update iai-callgrind",
+                            "iai-callgrind-runner ({0}) is newer than iai-callgrind ({1}). Please \
+                             update iai-callgrind to '{1}' in your Cargo.toml file",
```
Comment (Joining7943) [inline iai-callgrind-runner/src/main.rs:80]: iai-callgrind needs to be updated to the newer runner version, which is `{0}`. So, this needs to be `... update iai-callgrind to '{0}' in your ...`

**Review by Joining7943 — CHANGES_REQUESTED** [review]
Please don't resolve reviews. I'll resolve them, when everything's alright.

File: iai-callgrind-runner/src/main.rs (hunk 1 above)
Comment (Joining7943) [reply iai-callgrind-runner/src/main.rs]: Thanks :)

**Pull request merged**